internal/armada/repository/queue.go

```diff
@@ -11,6 +11,20 @@ import (
 
 const queueHashKey = "Queue"
 
+// TODO These errors should be of a specific error type, e.g.:
+// type ErrQueueNotFound struct {
+// 	queueName string
+// }
+// func (err *ErrQueueNotFound) String() {
```

Comment:
I think the interface is `Error()`, not `String()`

Reply:
You're right. That's an `Error()` on my part.